app/plugin_api.go

```diff
@@ -34,6 +34,32 @@ func NewPluginAPI(a *App, manifest *model.Manifest) *PluginAPI {
 	}
 }
 
+func (api *PluginAPI) applyDefaultConfigValues(pluginConfig map[string]interface{}) {
+	initialConfig := make(map[string]interface{})
+
+	if api.manifest.SettingsSchema != nil {
```

Comment:
Suggest we invert and return early if this is `nil`, since there's nothing for the method to do.